src/index.ts

```diff
@@ -1 +1 @@
-export default {}
+export {VersionDetail, default} from './commands/version'
```

Comment:
Do you mean to export the Version command here with `default`? How will that be used later?

Reply:
Yes it's going to be used in the sfdx and sf CLIs.

Reply:
Got'cha, might I suggest using an `as` to be a little bit more explicit?
```suggestion
export {VersionDetail, default as VersionCommand} from './commands/version'
```